processor/batchprocessor/metrics.go

```diff
@@ -57,10 +68,21 @@ func MetricViews() []*view.View {
 		Aggregation: view.Distribution(10, 25, 50, 75, 100, 250, 500, 750, 1000, 2000, 3000, 4000, 5000, 6000, 7000, 8000, 9000, 10000, 20000, 30000, 50000, 100000),
 	}
 
+	distributionBatchSendSizeBytesView := &view.View{
+		Name:        statBatchSendSizeBytes.Name(),
+		Measure:     statBatchSendSizeBytes,
+		Description: statBatchSendSizeBytes.Description(),
+		TagKeys:     tagKeys,
```

Comment:
@bogdandrutu I am not sure if I got this right. 

The `processor.MetricTagKeys(level)` based on the level adds `service` and `processor` tags. Whereas the `processorTagKeys` contains only the `processor` tag. 

Shouldn't we use `tagKeys` for all views?

Reply:
when you record the size you record per service? It does not seem that you do, so you should use `processorTagKeys`

Reply:
Done, removed